lib/slim_lint/linter/trailing_blank_lines.rb

```diff
@@ -0,0 +1,18 @@
+module SlimLint
+  # Checks for trailing whitespace.
+  class Linter::TrailingBlankLines < Linter
+    include LinterRegistry
+
+    on_start do |_sexp|
+      dummy_node = Struct.new(:line)
+
+      if !document.source.end_with?("\n")
+        report_lint(dummy_node.new(document.source_lines.size),
+                    'No blank line in the end of file')
+      elsif document.source.lines.last.blank?
+        report_lint(dummy_node.new(document.source.lines.size),
+                    'Multiple empty lines in the end of file')
```

Comment:
This one checks if line is blank (can be not empty but contain only spaces). I don't see this cop you mentioned in current rubocop repo. Seems like it was merged into https://github.com/bbatsov/rubocop/blob/master/lib/rubocop/cop/layout/trailing_blank_lines.rb#L6